### PR TITLE
WINDUP-694 adding tags to rules and fixing formatting when tab is use…

### PR DESCRIPTION
**Pull request by mareknovotny**
…d instead of spaces

### CODE REVIEW

File: rules-reviewed/jboss-eap/glassfish/xml-glassfish.windup.xml [hunk 1]
```diff
@@ -24,13 +24,15 @@
...
-                    <tag>configuration</tag>
-		    </classification>
-                </iteration>
+               <iteration>
+                  <classification title="Glassfish Web Descriptor" effort="3" severity="mandatory">
+                     <description> Glassfish specific web descriptor file that is by convention named glassfish-web.xml. It is used to configure web application properties likes cache, session configuration, JNDI name binding etc.</description>
+                     <link
+                        href="https://access.redhat.com/documentation/en-US/JBoss_Enterprise_Application_Platform/6.2/html/Security_Guide/jboss-webxml_Configuration_Reference.html"
+                        title="JBoss EAP web descriptor" />
+                     <tag>configuration</tag>
```
Comment (jsight) [inline rules-reviewed/jboss-eap/glassfish/xml-glassfish.windup.xml]: Shouldn't this have glassfish as a tag as well?

File: rules-reviewed/jboss-eap/glassfish/xml-glassfish.windup.xml [hunk 2]
```diff
@@ -41,10 +43,10 @@
             <perform>
                 <iteration>
                     <classification title="Glassfish EJB Descriptor" effort="3" severity="mandatory">
-			<description> Glassfish specific ejb configuration file used to configure EJBs, map them to jndi names etc.</description>
-		    <link href="https://access.redhat.com/documentation/en-US/JBoss_Enterprise_Application_Platform/6/html/Development_Guide/jboss-ejb3xml_Deployment_Descriptor_Reference.html" title="JBoss-ejb3.xml Deployment Descriptor Reference"/>
-                    <tag>configuration</tag>
-		    </classification>
+                        <description> Glassfish specific ejb configuration file used to configure EJBs, map them to jndi names etc.</description>
+                        <link href="https://access.redhat.com/documentation/en-US/JBoss_Enterprise_Application_Platform/6/html/Development_Guide/jboss-ejb3xml_Deployment_Descriptor_Reference.html" title="JBoss-ejb3.xml Deployment Descriptor Reference"/>
+                        <tag>configuration</tag>
```
Comment (jsight) [inline rules-reviewed/jboss-eap/glassfish/xml-glassfish.windup.xml]: same here

File: rules-reviewed/jboss-eap/glassfish/xml-glassfish.windup.xml [hunk 3]
```diff
@@ -56,10 +58,10 @@
             <perform>
                 <iteration>
                     <classification title="Glassfish Application EAR configuration file" effort="3" severity="optional">
-			<description> Glassfish-application  is a Glassfish specific EAR configuration file used to configure security roles, url addresses, JNDI binding etc. This file is glassfish specific and needs to be migrated to application.xml or jboss-app.xml.</description>
-			<link href="http://www.jboss.org/schema/jbossas/jboss-app_7_0.xsd" title="JBoss application configuration xsd file"/>
+                        <description> Glassfish-application  is a Glassfish specific EAR configuration file used to configure security roles, url addresses, JNDI binding etc. This file is glassfish specific and needs to be migrated to application.xml or jboss-app.xml.</description>
+                        <link href="http://www.jboss.org/schema/jbossas/jboss-app_7_0.xsd" title="JBoss application configuration xsd file"/>
                         <tag>configuration</tag>
```
Comment (jsight) [inline rules-reviewed/jboss-eap/glassfish/xml-glassfish.windup.xml]: and here

File: rules-reviewed/fuse/sonicesb/xml-sonic-esb.windup.xml [hunk 4]
```diff
@@ -22,7 +22,9 @@
                 </xmlfile>
             </when>
             <perform>
-                <classification title="SonicESB Service Configuration" effort="4"/>
+                <classification title="SonicESB Service Configuration" effort="4">
+                    <tag>SonicESB</tag>
```
Comment (jsight) [inline rules-reviewed/fuse/sonicesb/xml-sonic-esb.windup.xml]: I think all lower-case would be good, in order to match the others.

**Comment by jsight** [comment]
Just a few comments and suggestions, but otherwise this is good to go.

File: rules-reviewed/jboss-eap/glassfish/xml-glassfish.windup.xml (hunk 1 above)
Comment (mareknovotny) [inline rules-reviewed/jboss-eap/glassfish/xml-glassfish.windup.xml]: not sure as metadata has got glassfish tag, I thought it is added as global tag, isn't it?

File: rules-reviewed/fuse/sonicesb/xml-sonic-esb.windup.xml (hunk 4 above)
Comment (mareknovotny) [inline rules-reviewed/fuse/sonicesb/xml-sonic-esb.windup.xml]: well, if we do prefer all lowercase I will change all others too like "Apache Camel" to "camel" WDYT?

Comment (jsight) [inline rules-reviewed/fuse/sonicesb/xml-sonic-esb.windup.xml]: I think that would be great. :)

**Comment by mareknovotny** [comment]
i have pushed updated commit from our discussion.

**Pull request merged**